utils/build.gradle

```diff
@@ -22,4 +23,28 @@ dependencies {
     testImplementation "junit:junit:${junitVersion}"
     testImplementation "com.google.truth:truth:${truthVersion}"
     testImplementation "org.mockito:mockito-core:${mockitoVersion}"
+    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8:$kotlin_version"
+}
+
+buildscript {
```

Comment:
Do we use kotlin from parent build.gradle? There are some modules have used it.

Reply:
oops, yeah you are right. I don't need to define this

All I need to do is apply the plugin because this all is already set up in the `build.gradle` parent